holidays/countries/kuwait.py

```diff
@@ -0,0 +1,77 @@
+#  python-holidays
+#  ---------------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/dr-prodigy/python-holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Kuwait(HolidayBase, InternationalHolidays, IslamicHolidays):
+    """
+    References:
+        - https://en.wikipedia.org/wiki/2024_in_Kuwait
+        - https://www.officeholidays.com/countries/kuwait
+        - https://www.timeanddate.com/holidays/kuwait/2024
+    """
+
+    country = "KW"
+    default_language = "ar"
+    # %s (estimated).
+    estimated_label = tr("(تقدير) %s")
+    supported_languages = ("ar", "en_US")
+
+    def __init__(self, *args, **kwargs):
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("رأس السنة الميلادية"))
+
+        # Isra and Miraj.
```

Comment:
Maybe it would be better to place Islamic holidays after Gregorian calendar holidays, and in order according to the Islamic calendar: Islamic New Year, Prophet's Birthday, Isra and Miraj, Eid al-Fitr, Arafat, Eid al-Adha?

Reply:
Hello. Will this code be modified soon? I'm waiting for Kuwait holidays~~

Reply:
@KJhellico done, PTAL

@jamjam0109 this'll be included to the next PH release (~2 weeks)